### ISSUE TITLE
Can't create first admin user (最初の管理者ユーザーを作成できない)

### ISSUE THREAD

**Issue by s2terminal** [weseek/growi#810]
I can't create first admin user, so there is no way to create an administrator user.

Environment
------------

### Host

| item     | version |
| ---      | --- |
|OS        | Windows 10 Pro 1809 |
|GROWI     |3.3.5|
|node.js   |8.15.0|
|npm       |6.4.1|
|Using Docker|yes|
|Using [growi-docker-compose][growi-docker-compose]|yes|

[growi-docker-compose]: https://github.com/weseek/growi-docker-compose

*(Accessing https://{GROWI_HOST}/admin helps you to fill in above versions)*

### Client

| item     | version |
| ---      | --- |
|OS        | Windows 10 Pro 1809 |
|browser   |Google Chrome 71.0|

How to reproduce? (再現手順)
---------------------------

1. according to the [setup procedure](https://github.com/weseek/growi-docker-compose#start), I executed `git clone`, `cd`, `docker-compose up`
1. I accessed  http://localhost:3000/

What happens? (症状)
---------------------

- When I accessed http://localhost:3000/installer , redirect to `/login` and displayed "Application already installed."
- I created a new user from "新規登録はこちら", but I don't have admin access. So I can't create admin account.

What is the expected result? (期待される動作)
-------------------------------------------

- If there are no users, I can access `/installer`
- Or I can create an administrator user in some other way.

Note
----

- Before creating a new user, there are no users in MongoDB `users` collection.
    ```bash
    # mongo admin
    > use growi
    > db.users.find().count()
    0
    ```
- After creating a new user from "新規登録はこちら", I could set admin access manually with the following query.
    ```bash
    # mongo admin
    > use growi
    > db.users.update( { admin : false }, { $set: { admin : true }})
    WriteResult({ "nMatched" : 1, "nUpserted" : 0, "nModified" : 1 })
    ```

**Comment by utsushiiro** [comment]
@s2terminal  Thank you for your report. I'll fix it.

**Comment by yuki-takei** [comment]
This issue has fixed on v3.3.6.
